_includes/configuration/advtable-value-series.md

```diff
@@ -0,0 +1,82 @@
+## `advtable_value_series`
+{{ site.requires_5_8 }}
+
+{% if pluginname != "Advanced Tables" %}
+> **Note**: The `advtable_value_series` option requires the Advanced Tables plugin.
+{% endif %}
+
+The `advtable_value_series` option allows values series to be configured that can be used to populate cells in a table.
+
+**Type:** `Object`
+
+**Default Value:**
+ - `numbers` - a natural number series.
+ - `alpha` - a base26 letter series.
```

Comment:
I feel like adding the info that it's base26 makes it more confusing. Could we just call it "an alphabetic series" or something like that (maybe with a note that it uses the English alphabet)?